tests/helpers.js

```diff
@@ -116,7 +121,18 @@ class FuzzTest {
 		for (const dictionary of this.dictionaries) {
 			options.push("-dict=" + dictionary);
 		}
-		this.runTest("npx", options, { ...process.env });
+		if (!sync) {
+			return this.runTestAsync("npx", options, { ...process.env }).then(
+				(result) => {
```

Comment:
These handles don't seem to do anything.

Reply:
It was doing something unnecessary and had a bug. I removed it and updated the tests to expect a promise rejected with the error exit code.